01-node-tutorial/answers/globals.js

```diff
@@ -0,0 +1,4 @@
+MY_VAR = "Hi there!";
```

Comment:
The instructions may not have been as clear as they could have been. Here, you actually do not need this statement: `MY_VAR = "Hi there"`.

In the assignment, there is an instruction for globals.js that reads: 
"Set an environment variable with the following command: `export MY_VAR="Hi there!"`"

This command should actually be done in your CLI or integrated terminal before running `node globals`

Reply:
Got it!